app/uploaders/disability_claim_document_uploader.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+class DisabilityClaimDocumentUploader < CarrierWave::Uploader::Base
+  MAX_FILE_SIZE = 25.megabytes
```

Comment:
EVSS gave us the 25 MB limit, but if there is an NSOC limit as well we could create a base uploader with that validation